src/pymovements/plotting/traceplot.py

```diff
@@ -42,7 +42,7 @@
 def traceplot(
         gaze: Gaze,
         position_column: str = 'pixel',
-        cval: np.ndarray | None = None,  # pragma: no cover
+        cval: np.ndarray | None = None,
```

Comment:
this must've been a strange copypaste error from a few lines above. don't know how this happened.